.github/workflows/maven.yml

```diff
@@ -21,13 +21,39 @@ on: [push, pull_request]
 
 jobs:
   build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 8
```

Comment:
Use 11 here

Reply:
Can do. Any particular reason though?

Reply:
Decrease usage of Java 8 😜 - 11 should be just faster. Personally i would use 17